ui/lib/etherscan-prefix-for-network.js

```diff
@@ -8,37 +8,19 @@ import * as networkEnums from '../../app/scripts/controllers/network/enums'
  */
 export function getEtherscanNetworkPrefix (networkId) {
   switch (networkId) {
+    case networkEnums.ROPSTEN_CHAIN_ID:
```

Comment:
I think we also need a case for `networkEnums.ROPSTEN` to replace the functionality of `getEtherScanNetworkIdentifier`.

Reply:
do we need this case just for `ROPSTEN`, or for the other networks as well?

Reply:
Sorry, I didn't specify that. It must be for all networks.

Reply:
Updated.